include/deal.II/numerics/data_out_dof_data.templates.h

```diff
@@ -1833,40 +1903,68 @@ DataOut_DoFData<DoFHandlerType, patch_dim, patch_space_dim>::
               // ensure that there is a continuous number of next
               // space_dim*space_dim components that all deal with tensors
               Assert(
-                i + size <= (*d)->n_output_variables,
+                i + size <= input_data->n_output_variables,
                 Exceptions::DataOutImplementation::ExcInvalidTensorDeclaration(
-                  i, (*d)->names[i]));
+                  i, input_data->names[i]));
               for (unsigned int dd = 1; dd < size; ++dd)
                 Assert(
-                  (*d)->data_component_interpretation[i + dd] ==
+                  input_data->data_component_interpretation[i + dd] ==
                     DataComponentInterpretation::component_is_part_of_tensor,
                   Exceptions::DataOutImplementation::
-                    ExcInvalidTensorDeclaration(i, (*d)->names[i]));
+                    ExcInvalidTensorDeclaration(i, input_data->names[i]));
 
-              // all seems alright, so figure out whether there is a common name
-              // to these components. if not, leave the name empty and let the
-              // output format writer decide what to do here
-              std::string name = (*d)->names[i];
+              // all seems alright, so figure out whether there is a common
```

Comment:
```suggestion
              // all seems right, so figure out whether there is a common
```
If I remember correctly, you made that change in the case above.